src/test/test-notify-recv.c

```diff
@@ -25,13 +27,19 @@ static void context_done(Context *c) {
 
 static int on_recv(sd_event_source *s, int fd, uint32_t revents, void *userdata) {
         Context *c = ASSERT_PTR(userdata);
+        struct ucred ucred = UCRED_INVALID;
```

Comment:
Please move the declaration after `_cleanup_strv_free_ char **l = NULL;`.
Also, the initialization is not necessary. Please drop.